pages/components/Game/index.js

```diff
@@ -0,0 +1,223 @@
+    if (finalScore > highScore) {
+      SetHighScore(finalScore)
+    }
+  }
+
+  const clearState = () => {
+    setGame({ ...initialvalue, scoreList: game.scoreList });
+    setResetScore(true);
+  };
+
+  const formatTime = (time) => {
+    time = parseInt(time);
+    const minutes = Math.floor(time / 60);
+
+    let seconds = time % 60;
+
+    if (seconds < 10) {
+      seconds = `0${seconds}`;
+    }
+
+    return `${minutes}:${seconds}`;
+  }
+
+  if (!game.isGameOver) {
+    gameControl = (
+      <Timer formatTime={formatTime} onGameover={onGameover} arrayofWords={arrayofWords} setWords={setWords}/>
+    );
+  }
+
+  else {
+    gameControl = (
+      <div className="text-center text-white">
+        <h1>GAME OVER!</h1>
+        <br></br>
+        <h1>Your Score</h1>
+        <br></br>
+        <h1>{formatTime(game.finalscore)}</h1>
+        <br></br>
+        {newHighScore}
+        <br></br>
+        <button className="btn-play-again" onClick={() => clearState()}>
+          <img className="icon-play-again" src="/reload.png" alt="Reload" width="40" height="40" />
+          PLAY AGAIN
+</button>
+        <style jsx>{`
+ .btn-play-again {
+  border: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  color: #ff5155;
+ }
+
+ .icon-play-again{
+   max-width: 2.2em;
+   max-height: 2.2em;
+   margin-right:20px;
+   padding-bottom: 10px;
+ }
+`}</style>
+      </div>
+    )
+  }
+
+  useEffect(()=>{
+    if(!(name || level)){
+      router.push('/components/Login')
+    }
+  },[name,level])
+
+  return (
+    <React.Fragment>
+      <div className="row">
+        <div className="col-sm-9">
+          <h3 className="text text-heading">
+            <img className="icon-player" src="/person.png" alt="Player" width="40" height="40" />
+            {name}</h3>
+          <br />
+          <h3 className="text text-heading">
+            <img className="icon-player" src="/gamepad.png" alt="GamePad" width="40" height="40" />
+            LEVEL: {level}</h3>
+        </div>
+        <div className="col-sm-3">
+          <h3 className="text text-heading">fast fingers</h3>
+          <Score formatTime={formatTime} isGameOver={game.isGameOver} setFinalScore={setFinalScore} reset={resetScore} setReset={setResetScore} />
+        </div>
+      </div>
+      <div className="row mt-5">
+        <div className="col-md-3 mb-3">
+          <div className="text-center">
+            <div className="scores-box">
+              <div className="mt-2">
+                <h5 className="text mb-2">SCORE BOARD</h5>
+                <hr />
+                {getScoreList()}
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6">{gameControl}</div>
+      </div>
+      <div className="row">
+          <div className="col-md-3">
+            <div className="start-game-element">
+              <button className="btn-quit-game" onClick={()=>router.back()}>
```

Comment:
you can make this <Link /> component https://nextjs.org/docs/api-reference/next/link